index.rst

```diff
@@ -6,25 +6,251 @@
+A catalog of clean surrounding stars is required to perform focus and alignment with the AuxTel observation. It is anticipated that rebuilding star catalog is required when new criteria should be applied on the selection of stars. This technote is to describe the notebook that creates new catalog based on queries from Tycho-2 and HD catalog. This notebook can be found on this github repo `https://github.com/lsst-sitcom/sitcomtn-058/tree/main/_static <https://github.com/lsst-sitcom/sitcomtn-058/tree/main/_static>`__.
+
+Generating a star catalog for the Auxtel observation
+====================================================
+
+
+The following description is about the Juypter notebook to generate the star catalog with required selection criteria. 
+See also details about `focusing, center, and absorb pointing offsets at AuxTel <https://obs-ops.lsst.io/Nighttime-Operations/Auxiliary-Telescope/AT-On-Sky/Focus-center-absorbPointingOffsets.html>`__.
+
+
+
+Create a notebook with the Tycho-2 catalogue
+--------------------------------------------
+First, a list of stars from Vizier query of  
+`Tycho-2 catalogue <http://vizier.cds.unistra.fr/viz-bin/VizieR-3?-source=I/259/tyc2>`__ [Tycho2]_
+The range of :abbr:`RA (Right ascension)`, :abbr:`Decl. (Declination)`, and :math:`{V_{T}}`.
+
+- ``file``: ``string``, the json file name of Tycho-2 query file. 
+
+- ``RAmdeg``: ``string``, mean Right Asc, ICRS, epoch=J2000 with logical operator
+
+- ``Demdeg``: ``string``, mean Decl, ICRS, at epoch=J2000 with logical operator
+
+- ``VTmag``: ``string``, Tycho-2 VT magnitude with logical operator
+
+The default selection criteria is all stars with ``DEmdeg = '<10.0'`` and ``VTmag = '<10'`` and the name of output file is ``file = './cwfs_tycho2_stars.pd'``.
+
+Cutting Selection Criteria
+--------------------------
+
+If this is the first time using this Jupyter notebook, Tycho-2 catalog with constraints will be quried from Vizier and save the list of all stars with constraints into json file ``file = './cwfs_tycho2_stars.pd'``. If the file already exists in local, it reads the existing file.
+
+.. code-block:: py
+   
+    if os.path.exists(file):
+        print('Read existing Vizier Catalog in Local'+file)
+        tycho_star_all = pd.read_json(file)
+    else :
+        print('Read Vizier Catalog I/259/tyc2 ')
+        target_stars = Vizier.query_constraints(catalog='I/259/tyc2', DEmdeg=DEmdeg, VTmag=VTmag)[0]
+        result = target_stars.to_pandas().to_json(file)
+
+In order to execlude targets contaminated from bright nearby star, following criteria are further applied to the Tycho-2 catalog:
+
+- Include stars with 6 < V < 8 
+
+- Elimiate stars with
+
+  - very bright (V < 10 mag) nearby stars within 1 arcmin. radius 
+  - bright (V < 6 mag) within 7 arcmin radius
+  - extremely nearby stars (V < 3mag) within 20 arcmin radius  
+
+- Exclude stars with higher proper motion (> 500 mas/yr) 
+
+These constraints can be also changed with following parameters:
```

Comment:
So this is fine to put here, but it's not really necessary. I think you explained it well enough in the notebook itself.